packages/nextjs/src/server/withClerkMiddleware.ts

```diff
@@ -41,6 +41,9 @@ export const decorateResponseWithObservabilityHeaders = (res: NextResponse, requ
   requestState.status && res.headers.set(constants.Headers.AuthStatus, encodeURIComponent(requestState.status));
 };
 
+/**
+ * @deprecated Use authMiddleware instead https://clerk.com/docs/nextjs/middleware
```

Comment:
This was the main exported api, so let's update the comment to
```suggestion
 * @deprecated withClerkMiddleware has been deprecated in favor of `authMiddleware` instead. For more details, consult the middleware docs: https://clerk.com/docs/nextjs/middleware
```

Reply:
Done